src/rsa/verification.rs

```diff
@@ -89,10 +101,8 @@ pub fn verify_rsa(params: &RSAParameters,
                   (n, e): (untrusted::Input, untrusted::Input),
                   msg: untrusted::Input, signature: untrusted::Input)
                   -> Result<(), error::Unspecified> {
-    const MAX_BITS: usize = 8192;
-
     let signature = signature.as_slice_less_safe();
-    let mut decoded = [0u8; (MAX_BITS + 7) / 8];
+    let mut decoded = [0u8; (PUBLIC_MODULUS_MAX_LEN + 7) / 8];
```

Comment:
I'll rename this variable to `PUBLIC_MODULUS_MAX_BITS` for clarity. I expect `*_LEN` to be in terms of array elements, which can't be bits.